### ISSUE TITLE
get_attr_string fails when attribute is missing rather than returning None

### ISSUE THREAD

**Issue by dlindsaye** [thierry-martinez/pyml#93]
My understanding is that `Py.Object.get_attr_string` should return `None` if the attribute is absent:

```
utop # let bool_ty = Py.Object.get_type Py.Bool.t;;
val bool_ty : Pytypes.pyobject = <abstr>

utop # Py.Object.get_attr_string bool_ty "dtype";;
Exception:
E (<class 'AttributeError'>, type object 'bool' has no attribute 'dtype')
Raised at Py.python_exception in file "py.ml", line 812, characters 6-31
Called from Py.option in file "py.ml", line 902, characters 6-20
```
And thanks for all your work!

**Comment by thierry-martinez** [comment]
Nice catch! Should be fixed by #94.

**Comment by thierry-martinez** [comment]
Thanks a lot!
